contracts/swaps/connectors/testnet/SwapsImplLocal.sol

```diff
@@ -99,14 +99,16 @@ contract SwapsImplLocal is State, ISwapsImpl {
      * @param destTokenAddress The address of the destiny tokens.
      * @param sourceTokenAmount The amount of source tokens.
      * @param unused Fourth parameter ignored.
+     * @param defaultPath defaultPath for swap.
      *
      * @return precision The expected return.
      * */
     function internalExpectedReturn(
         address sourceTokenAddress,
         address destTokenAddress,
         uint256 sourceTokenAmount,
-        address unused
+        address unused,
```

Comment:
do we need to keep this param?

Reply:
Yes, to override the inheritted function from the parent